.github/workflows/ci.yml

```diff
@@ -9,6 +9,9 @@ jobs:
     strategy:
       matrix:
         version:
+          - '1.0'
+          - '1.1'
+          - '1.2'
```

Comment:
We also need to run CI on windows and mac and x86. Should we maybe just put the entire package under pkg butler management, so that it is aligned with how we handle all the other packages?

Reply:
sounds alright to me